src/scrabble.js

```diff
@@ -1,5 +1,137 @@
-function scrabble() {
+function isAlpha(str) {
+  return (
+    /^[A-Za-z]*$/.test(str) ||
+    str === '{' ||
+    str === '}' ||
+    str === '[' ||
+    str === ']'
+  )
+}
+
+function getLettersInBracket(fromIndex, toIndex, scrabbleArray) {
+  return scrabbleArray.slice(fromIndex, toIndex)
+}
 
+function getTotalScore(k, scrabbleArray) {
+  if (
+    scrabbleArray[k] === 'a' ||
+    scrabbleArray[k] === 'e' ||
+    scrabbleArray[k] === 'i' ||
+    scrabbleArray[k] === 'o' ||
+    scrabbleArray[k] === 'l' ||
+    scrabbleArray[k] === 'n' ||
+    scrabbleArray[k] === 'r' ||
+    scrabbleArray[k] === 'u' ||
+    scrabbleArray[k] === 's' ||
+    scrabbleArray[k] === 't'
+  ) {
+    console.log('here')
+    return 1
+  }
+  if (scrabbleArray[k] === 'd' || scrabbleArray[k] === 'g') {
+    return 2
+  }
+  if (
+    scrabbleArray[k] === 'b' ||
+    scrabbleArray[k] === 'c' ||
+    scrabbleArray[k] === 'm' ||
+    scrabbleArray[k] === 'p'
+  ) {
+    return 3
+  }
+  if (
+    scrabbleArray[k] === 'f' ||
+    scrabbleArray[k] === 'h' ||
+    scrabbleArray[k] === 'v' ||
+    scrabbleArray[k] === 'w' ||
+    scrabbleArray[k] === 'y'
+  ) {
+    return 4
+  }
+  if (scrabbleArray[k] === 'k') {
+    return 5
+  }
+  if (scrabbleArray[k] === 'j' || scrabbleArray[k] === 'x') {
+    return 8
+  }
+  if (scrabbleArray[k] === 'q' || scrabbleArray[k] === 'z') {
+    return 10
+  }
+  return 0
 }
```

Comment:
Nice way to organise the reference letter scores, and give a return of 0 if an invalid character in given at the same time :)

Reply:
Thanks!